### ISSUE TITLE
Excessive CAS

### ISSUE THREAD

**Issue by jnoelc** [Savage-Game-Design/Mike-Force#110]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is: there is excessive CAS and AI support available, leading to the majority of enemy bieng wiped out by the green hornets, and some extreme cases of friendly fire.

**Describe the solution you'd like**
Updated after thought: have a dedicated FAC, who is a member of the green hornets. (Make it a role specific to them). Air assets are initially transport only.

This role decides how to spend a "currency", which slowly increases in proportion to the amount of people in game.

This currency can be spent on AI air strikes, or on transport or CAS assets spawned at the the AF.

The second part of his role is designating CAS target zones. GH pilots gain points/rank by hitting targets in these zones. They loose points for hitting outside these zones, and are excessivly penalised for friendly fire.

There are max 2 of these roles for redundancy.

Make Arty Fwd controller a separate role that only MF can do, using current cool Down model.

**Describe alternatives you've considered**

Apply the zoning principle in another way, where all players can use it, or make it part of the radio operater role as it is currently

**Additional context**

**Comment by foofski** [comment]
I think the key to solving the friction between GH and MF re: CAS in communication rather than separation - the two groups can only talk to each other in sidechat or via map markers (unless TFAR is utilised). The task request system is a useful bridge for this gap - but probably too clunky to use in the heat of battle.

The source(s) of the MF-GH friction are a mix of a) not knowing what CAS is about to hit; b) being the victim of friendly fire; and/or c) not being able to get in on the action. Anxiety, frustration, and helplessness ensues. For this reason, any FAC-style role should be a part of MF rather than GH.

Lastly, I think the bar to flying the more dangerous CAS aircraft (e.g. F4, Cobra, Gunships etc) is too low. Perhaps the ranking / XP  system could be utilised in some way (like it is for the arsenal?).

**Comment by Spoffy** [comment]
Agreed that this is a major issue that needs addressing - your suggestions will be taken on-board, as we come up with a solution to this! 🙂

**Comment by ewan-000** [comment]
I feel that you need tasks that only infantry can complete. This will allow people to still fly as they like, without depriving infantry of a job.

1. remove site markers from map - current game play = "F4/arty bombs sites faster than infantry can reach them"
2. add intel mission of some kind which requires boots on ground (search dead for items, search village for box, talk to civ) on completion location of site is given
3. remove "weaken enemy" - current game play = "sit in fob watching helos nuke waves of enemy"
4. add "patrol AO" after "destroy supplies" task requires players to hit ground waypoints in order with ambush spawned at random one.

If this goes too far one way or the other, you can balance by adding GH or ACAV styled missions. "bring tank to location", "bomb airfield", "bridge river" etc

**Comment by LuizBarros99** [comment]
I believe that most cases of friendly fire are likely caused from bad communication, with the exception of inexperienced or malicious players.
With proper communication and having the player markers setting enabled, it should be easy to avoid friendly fire.

Now, something that could possibly help to avoid CAS players completely destroy enemy objective sites before infantry can reach them, would be to make their markers lesser accurate. Or possibly even consider entirely disabling almost every single one of these markers for Green Hornet players.

Another option would be to give the VC/NVA better air defenses, I already do have a suggestion about this. And it basically consists in complementing the ZPU-4 AAs with SA-2/S-75 sites (one per zone), and also have the possibility of NVA soldiers carrying either of the 9K32 Strela-2/2M launchers.
Basically this would give air defense systems with longer effective ranges, which could be useful against jets such as the F-4B/C Phantom, and F-100D. On top of making SEAD/DEAD sorties a possibility.

Finally, I think that adding MiG-19 interceptor(s) that are spawned to disrupt the player controlled CAS could also be an interesting option.
